### PR TITLE
Update site to solar

### PR DESCRIPTION
**Pull request by meganchoy**
Most to all major changes are complete. 

The only thing left to do - that is not a huge deal - is updating the silhouette on the footer

### CODE REVIEW

**Review by LukeHouge — APPROVED** [review]
Looks good, thanks for getting this updated!

I am going to edit the text in the timeline and update the sponsors, and then if you could take a look at the stats not loading it should be good to go.

File: public/contact.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+	<link rel="stylesheet" href="assets/css/main.css" />
+	<link rel="stylesheet" href="assets/css/bloop.css" />
+	<meta name="description"
+		content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the SpaceX Hyperloop Pod Competition">
+	<meta name="keywords" content="hyperloop,badgerloop, pod, madison, spacex, uw, uw-madison, engineering">
+	<link rel="shortcut icon" type="image/png" href="images/favicon.png" />
+	<meta property="og:title" content="Badgerloop" />
+	<meta property="og:image" content="images/logo.png" />
+	<meta property="og:description"
+		content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the SpaceX Hyperloop Pod Competition" />
```
Comment (EUdds) [inline public/contact.html:17]: Same

File: public/error.html [hunk 2]
```diff
@@ -15,11 +15,6 @@
 	<meta property="og:image" content="images/logo.png" />
 	<meta property="og:description"
 		content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the SpaceX Hyperloop Pod Competition" />
```
Comment (EUdds) [inline public/error.html:17]: Same

**Review by EUdds — CHANGES_REQUESTED** [review]

**Review by EUdds — APPROVED** [review]

File: public/about-badgerloop.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+<html>
+
+<head>
+    <title>About Badgerloop</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <link rel="stylesheet" href="assets/css/bloop.css" />
+    <link rel="stylesheet" href="assets/css/timeline.css" />
+    <meta name="description" content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the SpaceX Hyperloop Pod Competition">
```
[the remark this answers is not in the document]
Comment (meganchoy) [reply public/about-badgerloop.html]: see new commits

File: public/index.html [hunk 4]
```diff
@@ -3,23 +3,26 @@
...
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <link rel="stylesheet" href="assets/css/bloop.css" />
     <meta
       name="description"
-      content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the SpaceX Hyperloop Pod Competition"
+      content="Badgerloop is a student organization at the University of Wisconsin-Madison that competes in the American Solar Chalelnge"
```
Comment (LukeHouge) [inline public/index.html]: can you fix this spelling on all these meta tags?

**Pull request merged**